style.css

```diff
@@ -14,7 +16,10 @@ header {
   align-items: center;
   padding: 15px 24px;
   background-color: #fff;
-  font-family: 'Poppins', sans-serif;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 72px;
```

Comment:
- [ Required ] Please consider adding a `z-index: 1` CSS rule on the styling of your `header` section to make it always be on top of other elements when a user is scrolling the page. I experienced the screenshot below when browsing the page at the `About` section of the page. You can fix this by adding this CSS rule to your header styling.

```CSS
  z-index: 1;
```

## Your design
![Screenshot from 2023-01-19 12-36-16](https://user-images.githubusercontent.com/16392046/213407751-4c10ac31-f98a-4573-bd11-ec0ea7295dc1.png)